buildtest/schemas/examples/spack.schema.json/invalid/spack_load_invalid_example.yml

```diff
@@ -0,0 +1,16 @@
+buildspecs:
+  spack_load_invalid_example:
+    type: spack
+    executor: generic.local.bash
+    description: "Run spack load for m4 package and run tests"
+    tags: [spack]
+    spack:
+      root: $HOME/spack
+      load:
+        options: '--only package'
+        specs: [m4]
+      test:
+        run:
+          specs: ['m4', 1]
```

Comment:
remove the `1` from here instead you should put the `1` under `specs` section for `load`. You want to make test invalid and it needs to be something specific to the `load` section.